test/api.js

```diff
@@ -599,7 +601,8 @@ describe("API", async () => {
                 it(`${_method.toUpperCase()} ${path}: response status code should match one of the schema defined responses`, () => {
                     // HACK: allow HTTP 418 I am a teapot, for now   👇
```

Comment:
what does this mean, unclear here, but looks fine

Reply:
Hey, this was already in the legacy code, not added by me